seed_stage_based_messaging/utils.py

```diff
@@ -55,4 +56,13 @@ def get_available_metrics():
         available_metrics.append(
             "subscriptions.message_set.{}.total.last".format(messageset_name))
 
+    subscriptions = Subscription.objects.all()
+    for subscription in subscriptions:
+        lang_key = "subscriptions.language.{}.sum".format(subscription.lang)
+        if (lang_key not in available_metrics):
+            available_metrics.append(lang_key)
+            available_metrics.append(
+                "subscriptions.language.{}.total.last"
+                .format(subscription.lang))
```

Comment:
There shouldn't be any funny characters, but since this is just a normal char field, technically we can put anything in there. Maybe we should put the lang through the `normalise_metric_name` function to make sure it's normalised?